test/common/upstream/load_balancer_fuzz_base.cc

```diff
@@ -6,14 +6,30 @@ namespace Envoy {
 namespace Upstream {
 
 namespace {
-// TODO(zasweq): This will be relaxed in the future in order to fully represent the state space
-// possible within Load Balancing. In it's current state, it is too slow (particularly due to calls
-// to makeTestHost()) to scale up hosts. Once this is made more efficient, this number will be
-// increased.
-constexpr uint32_t MaxNumHostsPerPriorityLevel = 256;
+
+constexpr uint32_t MaxNumHostsPerPriorityLevel = 60000;
+
+// Helper function for converting repeated proto fields to byte vectors to pass into random subset
+std::vector<uint32_t>
+constructByteVectorForRandom(const Protobuf::RepeatedField<Protobuf::uint32>& random_bytestring) {
+  std::vector<uint32_t> random_bytestring_vector;
+  for (int i = 0; i < random_bytestring.size(); ++i) {
+    random_bytestring_vector.push_back(random_bytestring.at(i));
```

Comment:
what do you think about just using the fact that a repeated ptr field is a container:

`std::vector<uint32_t> random_vector(random_bytestring.begin(), random_bytestring.end());`

Reply:
Oh nice, great suggestion, thanks!